README.md

```diff
@@ -1,5 +1,102 @@
-# dlang-bot
-Dlang-bot for automated bugzilla, github, and trello references.
-Also cancels stale Travis-CI builds
+# Dlang-Bot
 
-![dlang-bot](public/img/dlang_bot.png)
+<img alt="dlang-bot" height="200px" src="public/img/dlang_bot.png" />
+
+Features
+--------
+
+- [Automated bugzilla, github, and trello references](#automated-references)
+- [Auto-merging approved & passing PRs](#auto-merge)
+- [Canceling stale Travis-CI builds](#canceling-stale)
+
+
+<a name="automated-references" />
+
+Automated references
+--------------------
+
+For example let's say you fixed [Issue 16582](https://issues.dlang.org/show_bug.cgi?id=16582)
+and make a PR for on GitHub.
+If one of your commits mentions the issue, e.g. like this Git commit message:
+
+```
+fix Issue 16582 - ParameterDefaults fails w/ scope parameter
+```
+
+The Dlang-Bot will do all the hard work of linking and referencing
+between Bugzilla, GitHub and Trello do for you.
+
+### GitHub
+
+It would comment on GitHub with list of all mentioned issues and link them to Bugzilla:
+
+<img alt="dlang-bot-github" height="120px" src="public/img/dlang_bot_github_comment.png" />
+
+The first column of the table is the current status of the referenced issue.
+
+### Bugzilla
+
+Once a PR has been merged (i.e. the commits got pushed to master or stable), the Dlang-Bot
+will automatically comment to Bugzilla with the regarding commit:
```

Comment:
> Maybe just put that section below the other ones and explain how Issues are referenced separately.

I moved the Bugzilla below the Trello section, but kept it in the "Issue referencing chapter" to avoid unnecessary confusion for the user.